src/libraries/System.IO.MemoryMappedFiles/tests/MemoryMappedViewAccessor.Tests.cs

```diff
@@ -97,6 +97,11 @@ public void ValidAccessLevelCombinations(MemoryMappedFileAccess mapAccess, Memor
                         // Containers and OSX with SIP enabled do not have execute permissions by default.
                         throw new SkipTestException("Insufficient execute permission.");
                     }
+                    else if ((OperatingSystem.IsIOS() && PlatformDetection.IsArm64Process) &&
```

Comment:
I think this isn't specific to arm64, it should happen on all iOS devices and simulators as well as on tvOS.

We can probably just add `|| OperatingSystem.IsIOS() || OperatingSystem.IsTvOS()` to the if above (and update the comment).